packaging/hip-targets.cmake

```diff
@@ -38,38 +38,54 @@ unset(_targetsNotDefined)
 unset(_expectedTargets)
 
 
-# The installation prefix configured by this project.
 if( DEFINED ENV{ROCM_PATH} )
-     set(_IMPORT_PREFIX "$ENV{ROCM_PATH}/hip")
-elseif(DEFINED ENV{HIP_PATH})
-     set(_IMPORT_PREFIX "$ENV{HIP_PATH}")
-else()
-     set(_IMPORT_PREFIX "/opt/rocm/hip")
+     set(ROCM_PATH "$ENV{ROCM_PATH}")
```

Comment:
There is no need to check for this here. Just use them both in the list of extra paths to search.